bijection-util/src/main/scala/com/twitter/bijection/twitter_util/UtilBijections.scala

```diff
@@ -122,6 +123,22 @@ trait UtilBijections {
       override def apply(pool: FuturePool) = new TwitterExecutionContext(pool)
       override def invert(context: ExecutionContext) = new ScalaFuturePool(context)
     }
+
+  object Owned {
```

Comment:
can we get some comments here to explain the differences? I guess in one case (shared?) you are copying the underlying buffer, but in the other you are not, which only makes it safe if you "own" the Buf/Array[Byte] in question. Is that right?